web-service-consumer/1.6/modules/ROOT/pages/web-service-consumer-studio.adoc

```diff
@@ -0,0 +1,276 @@
+
+Note that *WSDL location*, *Service*, and *Port* parameters are mandatory for the configuration to be valid.
+
+The following example gets a list of flights by airline. In Studio, configuring the global element looks as follows:
+
+.Web Service Consumer Global Element
+image::web-service-consumer-global-element.png[Web Service Consumer Global Element]
+
+In the *Configuration XML* tab, the XML looks as follows:
+[source,xml,linenums]
+----
+<wsc:config name="wsc">
+  <wsc:connection
+   wsdlLocation="http://ilt.mulesoft-training.com/essentials/delta?wsdl"
+   service="TicketServiceService"
+   port="TicketServicePort"/>
+</wsc:config>
+----
+
+
+[[configure-other-fields]]
+== Configure Consume Operation
+
+After you configure a global element for Web Service Consumer Connector, configure the Consume operation parameters, such as Operation, Message (Body, Headers, Attachments) and so on. Additionally, learn about the output of the Consume operation and metadata attributes used to dispatch messages.
+
+=== Parameters
+
+The Consume operation has two main parameters:
+
+* *Operation* +
+Defines which SOAP operation of the Web service to invoke. In design time, the parameter defines the input and output types for the Consume operation. The types will change depending on which operation you choose.
+
+* *Message* +
+A representation of the `SOAP:ENVELOP` composed of three optional parameters:
+** *Body* +
+The XML body to include in the SOAP message, with all the required parameters, or `null` if no parameters are required.
+** *Headers* +
+The XML headers to include in the SOAP message.
+** *Attachments* +
+The attachments to include in the SOAP request.
+
+To configure these parameters for the Consume operation, follow these steps: +
+
+. Select the name of the connector in the Studio canvas.
+. On the *General* section of the Consume operation define *Operation*, and in the *Message* section define *Body*, *Headers* and *Attachments*:
+
+.Configure Consume Operation
+image::web-service-consumer-configure-consume.png[Configure Consume Operation]
+
+In the *Configuration XML* tab, a basic configuration for the Consume operation looks as follows:
+[source,xml,linenums]
+----
+<wsc:consume config-ref="config" operation="addClients">
+    <wsc:message>
+        <wsc:body>#[payload]</wsc:body>
+    </wsc:message>
+</wsc:consume>
+----
+
+==== Body Parameter
+
+The `body` parameter is the main part of the SOAP message, and accepts embedded DataWeave scripts values so that you can construct the XML request without having a side effect on the message or having to use multiple components to create the request. The default value is `#[payload]` and expects that the incoming payload is the XML entity ready to ship to the service. +
```

Comment:
Important: The last sentence is unclear. I would break it into two sentences and say what expects the payload to be ready to ship to the service.